packages/core/src/stylable-resolver.ts

```diff
@@ -440,23 +436,6 @@ export class StylableResolver {
                     : meta.getClass(nameOrSymbol)
                 : nameOrSymbol;
 
-        const customSelector = isElement
```

Comment:
maybe we should throw if we encounter this?

Reply:
We have no internal flows that assumes this, so throwing without the `transformer` flag will probably break in case you have a custom-selector override in an unexpected manner.